test/manifold_test.cpp

```diff
@@ -160,6 +160,25 @@ TEST(Manifold, Decompose) {
   RelatedGL(manifolds, input);
 }
 
+TEST(Manifold, DecomposeProps) {
+  std::vector<Manifold> manifoldList;
+  manifoldList.emplace_back(WithPositionColors(Manifold::Tetrahedron()));
+  manifoldList.emplace_back(WithPositionColors(Manifold::Cube()));
+  manifoldList.emplace_back(WithPositionColors(Manifold::Sphere(1, 4)));
+  Manifold manifolds = Manifold::Compose(manifoldList);
+
+  ExpectMeshes(manifolds, {{8, 12, 3}, {6, 8, 3}, {4, 4, 3}});
+
+  std::vector<MeshGL> input;
+
+  for (const Manifold& manifold : manifoldList) {
+    auto mesh = manifold.GetMeshGL();
+    input.emplace_back(mesh);
```

Comment:
`WithPositionColors` *is* a `MeshGL` - why not create `input` at the same time you create `manifoldList`?

Reply:
OK